dnnsoftware/Dnn.Platform
# Issue 3703 - Removal of WebSlice Option

Fixes #3703

## Summary
The changes proposed here are to remove the WebSlice options from the Module Settings UI as well as the underlying tables, since it's no longer supported. 

## How to Test
- Do a clean install of DNN Platform and verify that the install completes without any errors.
- Upgrade an existing instance of DNN Platform and verify that the upgrade process completes without any errors
- Verify that the "Is a WebSlice" options have been removed from the Module Settings > Page Settings > Basic Settings section

Before this fix:
- Is a WebSlice options existed in Module Settings
![image](https://user-images.githubusercontent.com/13650025/122244198-adb80f00-ce92-11eb-9322-455742e2579e.png)
- Views, Tables, Stored Procedures and Constraints referenced the IsWebSlice column in the TabModules table
   - TabModules Table
      ![image](https://user-images.githubusercontent.com/13650025/122244973-50708d80-ce93-11eb-90b7-69e143e7b8e0.png)
   - TabModules Contraints
      ![image](https://user-images.githubusercontent.com/13650025/122245108-6ed68900-ce93-11eb-893f-8108e02172ac.png)
   - TabModules Indexes
      ![image](https://user-images.githubusercontent.com/13650025/122245396-afce9d80-ce93-11eb-9860-1db81d3ea42a.png)
   -  vw_TabModules
      ![image](https://user-images.githubusercontent.com/13650025/122245578-d68cd400-ce93-11eb-8448-97a2f7f63bb4.png)
   - SP: AddTabModule
     ![image](https://user-images.githubusercontent.com/13650025/122245813-0d62ea00-ce94-11eb-8270-79d608088444.png)
   - SP: UpdateTabModule
     ![image](https://user-images.githubusercontent.com/13650025/122246048-3be0c500-ce94-11eb-85e7-d2f5dd4af08e.png)





After this fix:
- No options exist for the WebSlice Options in the Module Settings UI
![image](https://user-images.githubusercontent.com/13650025/122244705-13a49680-ce93-11eb-8fbe-21729cfccfbb.png)

- The Tables, Views, SPs, Indexes, and Constraints have been updated/removed
